### PR TITLE
res_stasis.c: Add new type 'sdp_label' for bridge creation.

### PR DESCRIPTION
**Pull request by jjsearle**
Resolves: https://github.com/asterisk/asterisk/issues/91

### CODE REVIEW

**Comment by jcolp** [comment]
You will need to sign the contributor license agreement before this can be reviewed.

**Comment by jjsearle** [comment]
> You will need to sign the contributor license agreement before this can be reviewed.

I've signed the CLA

**Comment by AnasAlFalah** [comment]
Unsubscribe

Sent from my iPhone

On 17 May 2023, at 17:00, Joe Searle ***@***.***> wrote:

﻿

You will need to sign the contributor license agreement before this can be reviewed.

I've signed the CLA

File: rest-api/api-docs/bridges.json [hunk 1]
```diff
@@ -51,6 +51,15 @@
 							"required": false,
 							"allowMultiple": false,
 							"dataType": "string"
+						},
+						{
+							"name": "sendSDPLabel",
```
Comment (jcolp) [inline rest-api/api-docs/bridges.json:56]: I don't think this should be a field on the bridge create itself. I think it should be another property on the type. These define the characteristics of the bridge, and sending stream labels is a characteristic of it.

**Review by jcolp — CHANGES_REQUESTED** [review]
The commit message needs to be updated according to the [wiki](https://wiki.asterisk.org/wiki/display/AST/Commit+Messages).

A UserNote also needs to be added.

File: rest-api/api-docs/bridges.json (hunk 1 above)
Comment (jjsearle) [reply rest-api/api-docs/bridges.json:56]: Okay, I'll have a look into that and see if I can rework it.

**Comment by jjsearle** [comment]
I have reworked this and updated the commit message to match the wiki guidelines. Please could you review and add any further comments?

**Review by jcolp — CHANGES_REQUESTED** [review]
Commit message needs a tweak:

"will configure the bridge add" should be:
"will configure the bridge to add"

**Comment by jjsearle** [comment]
I have corrected the commit message, thank you.

**Review by jcolp — APPROVED** [review]

**Comment by gtjoseph** [comment]
If this PR applies to other branches, please add a comment with the appropriate "cherry-pick-to" headers as per the [Create a Pull Request](https://wiki.asterisk.org/wiki/display/AST/Code+Contribution#CodeContribution-CreateaPullRequest) process.  If you don't want it cherry-picked, please add a comment stating "No cherry-picks required" so we don't keep asking.

**Comment by jjsearle** [comment]
No cherry-picks required

**Comment by jjsearle** [comment]
> Does this not apply to 18?

I wasn't sure that adding a new feature would be accepted on 18. If it would I'll add the cherry-pick for 18 too.

**Comment by jcolp** [comment]
It would be.

**Review by gtjoseph — APPROVED** [review]

**Pull request merged**